server/balancer.go

```diff
@@ -603,6 +610,8 @@ func (h *balanceHotRegionScheduler) balanceByPeer(cluster *clusterInfo) (*Region
 	return nil, nil, nil
 }
 
+// selectDestStoreByPeer selects a target store hold the region of the source region.
+// We choose a target store based on the hot region number of this store and written bytes of this store.
```

Comment:
-> We choose a target store based on the hot region number and written bytes of this store.